### PR TITLE
MacOS install: "Deploying to iOS device" section rework

### LINKED ISSUE #1776: 'MacOS install' page issue-Deploying to physical iOS device
Page URL: https://flutter.io/docs/get-started/install/macos.html
Page source: https://github.com/flutter/website/tree/master/src/docs/get-started/install/macos.md

Specifically on 
https://flutter.io/docs/get-started/install/macos#deploy-to-ios-devices
it mentions  The following two steps are a temporary workaround to https://github.com/flutter/flutter/issues/22595
yet when i go there, the issue is closed.
**"temporary workaround" is not good for new people to see evaluating flutter if avoidable**
and 1.0 is so close...

thanks

### PR DESCRIPTION
**Pull request by chalin**
Fixes #1776 

cc @tvolkert 

---

screen shot 2018-12-10 at 13 53 22

### CODE REVIEW

File: src/docs/get-started/install/_ios-setup.md [hunk 1]
```diff
@@ -40,22 +40,35 @@ To prepare to run and test your Flutter app on the iOS simulator, follow these s
 
 To deploy your Flutter app to a physical iOS device, you’ll need some additional tools and an Apple account. You'll also need to set up physical device deployment in Xcode.
 
- 1. Install [homebrew](http://brew.sh/).
- 1. Open the terminal and run these commands to install the tools for deploying Flutter apps to
-    iOS devices.
+ 1. Install [homebrew](http://brew.sh).
```
Comment (kwalrath) [inline src/docs/get-started/install/_ios-setup.md]: http -> https?

File: src/docs/get-started/install/_ios-setup.md [hunk 2]
```diff
@@ -40,22 +40,35 @@ To prepare to run and test your Flutter app on the iOS simulator, follow these s
...
     $ brew link usbmuxd
     $ brew install --HEAD libimobiledevice
     $ brew install ideviceinstaller ios-deploy cocoapods
     $ pod setup
     ```
 
-   If any of these commands fails with an error, run `brew doctor` and follow the instructions
-   for resolving the issue.
+    {{site.alert.note}}
+      The first two steps are necessary as a temporary workaround until the next
```
Comment (kwalrath) [inline src/docs/get-started/install/_ios-setup.md]: steps -> commands above?

File: src/docs/get-started/install/_ios-setup.md (hunk 1 above)
Comment (chalin) [reply src/docs/get-started/install/_ios-setup.md]: Good catch. Fixed

File: src/docs/get-started/install/_ios-setup.md (hunk 2 above)
Comment (chalin) [reply src/docs/get-started/install/_ios-setup.md]: Yes, better. Done.

**Comment by chalin** [comment]
Changes made. Anything else?

**Pull request merged**